project/Dependencies.scala

```diff
@@ -0,0 +1,26 @@
+import DependencyVersions._
+import sbt._
+
+object Dependencies {
+
+  val globalExcludes = Seq(
+    SbtExclusionRule("log4j"),
+    SbtExclusionRule("log4j2"),
+    SbtExclusionRule("commons-logging")
+  )
+
+  val loggingDependencies = Seq(
+    "ch.qos.logback" % "logback-classic" % logbackVersion,
+    "com.github.danielwegener" % "logback-kafka-appender" % "0.1.0"
+  )
+
+  val jsonDependencies = Seq(
```

Comment:
I think we can group all this deps in one "projecDependences" as it is not needed to separate them